aptos-move/aptos-transactional-test-harness/Cargo.toml

```diff
@@ -20,7 +20,13 @@ serde = { version = "1.0.137", default-features = false }
 serde_json = "1.0.81"
 
 # Move dependencies
-move-deps = { path = "../move-deps", features = ["address32"] }
+move-binary-format = { workspace = true, features = [ "fuzzing" ] }
```

Comment:
Why features = [ "fuzzing" ] here but not in other places move-binary-format is included?

Reply:
It's a feature used only for testing purposes, and this is a testing crate.